.gitignore

```diff
@@ -15,3 +15,4 @@ renovate-0.0.0-semantic-release.tgz
 junit.xml
 /test-results
 renovate-schema.json
+test/datasource/nuget/_fixtures/obj
```

Comment:
This doesn't look related to this PR.

Reply:
yes, required if you have `ms-dotnettools.csharp` installed, because it restores nuget dependencies

Reply:
Yes, it's not directly related to this PR but I considered it too tiny for a dedicated PR